web_src/css/modules/animations.css

```diff
@@ -46,6 +46,11 @@ code.language-math.is-loading::after {
   height: 1.25rem;
 }
 
+#oauth2-login-navigator.is-loading::after {
```

Comment:
Not sure why it is needed. Add some comments?

Reply:
The default CSS rule has

```
  width: 4rem;
  height: 4rem;
```

Which is too big for this element. Maybe later we can try automatic sizing, but until then, this override is needed and already in place for other elements.

Reply:
I guess automatic sizing is impossible, but we could later add size variation classes to avoid having to alter CSS.

Reply:
It seems another overwriting game here. 😂 Nvm, just curious. Leave the framework improvement to the future.